stories/toolbar/toolbar.stories.js

```diff
@@ -1,13 +1,32 @@
 export default {
     title: 'Components/Toolbar',
     parameters: {
-        description: 'The toolbar enables the user to change the UI or trigger an action. For example, the toolbar allows the user to change views, manipulate data or objects, navigate to another page, perform generic actions, and so on. It is most commonly used to display buttons, labels, selects and various other input controls.',
+        description: `The toolbar enables the user to change the UI or trigger an action. For example, the user can change views, manipulate data or objects, navigate to another page, perform generic actions, and so on. This component is usually paired with buttons, which are always right-aligned.
+
+##Usage
+**Toolbar actions can be as follows:**
+
+- They can be independent of the current selection and not related to a specific item or object.
+- They can be specific to the current object (user selects one item).
+- They can apply to a set of items (user selects two or more items).
+- They can control the settings for parts of the UI content. For example, an action can affect all items in a **Table**.
+
+##Sizes
+There are two sizes of the toolbar that should be chosen based on the type of device and its screen width.
+
+Size&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Modifier class&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Screen width&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Device
+:---- | :-------- | :--------- | :-------
+Cozy | \`fd-toolbar--cozy\` | < 599 px | Mobile and small tablets
+Compact | \`fd-toolbar--compact\` | 600 px and above | Desktop
```

Comment:
`fd-toolbar--compact` modifier class is not used anywhere. It's the default one. If you want to use the toolbar in mobile mode than you need to add `fd-toolbar--cozy`

Reply:
Right, I'm not sure why I thought I saw that. Probably mixing up PRs!

Changed it to "Default"